distribution/lib/Standard/Base/0.0.0-dev/src/Data/Number/Extensions.enso

```diff
@@ -185,7 +185,7 @@ Number.format fmt = String.format fmt this
 
 ## ALIAS Range
```

Comment:
Should these method's be defined in the Range class?
Won't make much difference but easier to keep it managed together.

Reply:
I think you are right! Not sure how this plays in how extension methods will be approached after the types-redesign, but for now it makes more sense and if needed we can move it in the future.